lib/vsc/mympirun/option.py

```diff
@@ -120,6 +120,8 @@ def make_init(self):
 
             "stats": ("Set MPI statistics level", "int", "store", 0),
 
+            "timeout": ("Set timeout to check for output", "int", "store", 0)
```

Comment:
use a sensible default here, e.g. 600s (10m), and make it clear in the help message what the unit is (i.e. seconds)